action.yml

```
@@ -47,6 +51,10 @@ runs:
       run: |
         HUGO_FLAGS=" --invalidateCDN --maxDeletes -1"
         
+        if [[ "${{ inputs.environment }}" ]]; then
+          HUGO_FLAGS="$HUGO_FLAGS --environment ${{ inputs.environment }}"
```

Small thing to keep consistency:
```suggestion
          HUGO_FLAGS="$HUGO_FLAGS --environment=${{ inputs.environment }}"
```